assets/charts/components/headlamp/templates/deployment.yaml

```diff
@@ -6,9 +6,6 @@ metadata:
     {{- include "headlamp.labels" . | nindent 4 }}
```

Comment:
It seems commit 2cbc15ed7e1bc7f5cf294c5974ed2171760667c6 should be split and fixed up into previous comments.